electroncash/address.py

```diff
@@ -534,6 +541,8 @@ def to_full_string(self, fmt, *, net=None):
 
     def to_ui_string(self, *, net=None):
```

Comment:
Hmm. Despite the name of this function, I am afraid maybe some inner parts of the code not tied to UI rely on this to return a non-full address, sans prefix.  You may want to grep the entire codebase, including plugins, to make sure no invariants are broken by this change..

Reply:
The only plugin that I could find that used the function `to_ui_string()` directly was Ledger, but that was OK since it's only for printing.
I haven't found anything else that would break because of this change. Note that anyone calling to_ui_string() can't really know if the returned string is legacy or cashaddr either.

Reply:
Hmm ok thanks for the follow-up on that.